src/werkzeug/__init__.py

```diff
@@ -3,4 +3,4 @@
 from .wrappers import Request
 from .wrappers import Response
 
-__version__ = "2.0.0.dev"
+__version__ = "2.0.0rc1.dev"
```

Comment:
Based on the version standard this ought to be `2.0.0rc2.dev` to sort correctly

Reply:
Thanks.